src/lib.rs

```diff
@@ -261,17 +260,19 @@ mod arch {
 
                         let trace = match blondie::trace_command(command_builder, false) {
                             Err(err) => {
-                                eprintln!("{}: {:?}", BLONDIE_ERROR, err);
-                                exit(1);
+                                bail!("{}: {:?}", BLONDIE_ERROR, err);
```

Comment:
Want to inline the format arguments while you're at it?

Reply:
Done